services/catalog/setup.py

```diff
@@ -56,7 +68,7 @@ def read_reqs(reqs_path: Path):
         ],
     },
     include_package_data=True,
-    install_requires=install_requirements,
+    # install_requires=install_requirements,
```

Comment:
is this correct? should it stay commented?

Reply:
good catch! I missed that!